notebooks/official/matching_engine/similar_products_recommender_using_matching_engine.ipynb

```diff
@@ -0,0 +1,1683 @@
+{
```

Comment:
<p>Test query -&gt; Test the query. (Increase the font size)</p><p> </p><p>The split between 1. and 2. does not make sense</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='95'/>

Reply:
<p>updated</p>